### DISCUSSION TITLE
Can I plot feature importance in Sequential Feature Selection for Regression

### DISCUSSION THREAD

**Discussion by lewisscola** [rasbt/mlxtend#831] [Q&A]
Hi,

I'm using Sequential Feature Selection for Regression in mlxtend. The website is(Example 5): http://rasbt.github.io/mlxtend/user_guide/feature_selection/SequentialFeatureSelector/#example-3-visualizing-the-results-in-dataframes

My question is: can I plot the feature importance for each feature?

Thank you!

**Comment by rasbt** [comment]
Sorry that info is not available via the SFS. I.e., the SFS does not compute feature importance in a meaningful way (because the feature importance could change via each subset iteration, and it would be an unfair way to compute feature importances that way).

What you could do though is running SFS, selecting the features of interest, and then getting the feature importances via http://rasbt.github.io/mlxtend/user_guide/evaluate/feature_importance_permutation/

**Reply by lewisscola** [reply]
Thank you very much!
